blocks/carousel/carousel.css

```diff
@@ -0,0 +1,94 @@
+.carousel {
+  width: 100%;
+  margin-bottom: 2rem;
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+  
+
+.carousel>ul {
+  width: 70%;
+  display: flex;
+  flex-wrap: nowrap;
+  overflow-x: scroll;
+  gap: 2rem;
+  list-style: none;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+}
+
+.carousel>ul>li {
+  scroll-snap-align: start;
+  
+}
+
+.carousel>ul::-webkit-scrollbar {
+  display: none;
+}
+
+/* Specific card overrides for carousel - BEGIN */
+.carousel .card {
+  max-width: 28rem;
+  flex-shrink: 0;
+}
+
+.carousel .card .card-body {
+  margin-bottom: 0;
+  color: var(--text-color);
+  text-align: left;
+}
+
+/* Specific card overrides for carousel - END */
+
+.carousel .carousel-nav {
+  position: absolute;
+  background: none;
+  width: 100%;
+  max-width: 75%;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  top: 0;
+  
+}
+
+.carousel .carousel-nav button{
+  background: none;
+  display: flex;
+  align-items: center;
+  justify-content:center;
+  padding: 1rem;
+  z-index: 2;
+}
+
+.carousel .icon-arrow-right{
+  background-color: #f5f5f0;
+  border-radius: 50%;
+}
+
+.carousel .carousel-nav .icon svg {
+  fill: #989090;
+  width: 30px;
+  height: 30px;
+}
+
+.carousel .icon-arrow-right.left-arrow {
+  transform: rotate(180deg);
+}
+
+@media only screen and (min-width:701px) {
+  .carousel .card {
+    max-width: 12rem;
+  }
+
```

Comment:
And for the multiple cards, add a style to indicate how many cards to show:
.carousel.4-cards > li {
min-width: calc(100% / 4);
}

.carousel.5-cards > li {
min-width: calc(100% / 5);
}